atlasdb-api/src/main/java/com/palantir/atlasdb/config/LeaderConfig.java

```diff
@@ -85,6 +85,8 @@ protected final void check() {
 
     @Value.Derived
     public LockLeader whoIsTheLockLeader() {
-        return LockLeader.iAmTheLockLeader(lockCreator().equals(localServer()));
```

Comment:
It seemed a bit confusing, `iAmTheLockLeader` didn't conform to variable naming specs (needs two lowercase characters at the beginning), and I couldn't think of a better name.

After having a break from this PR, I can think of one now: `LockLeader.fromBoolean`. Do you think that's better?

I kind of like the inlined version though.